continuous_integration/travis/modules_test.sh

```diff
@@ -94,12 +94,3 @@ for f in spyderlib/*/*/*.py; do
         exit 1
     fi
 done
-
-
-# Spyderplugins
-for f in spyderplugins/widgets/*.py; do
```

Comment:
Nope, that does not cut it... it was not only that

Test wont run cause they do not run as standalone scripts, they only work when a spyder instance runs